docs/_sidebar.md

```diff
@@ -0,0 +1,12 @@
+- Usage
+    - [new DashPlatformProtocol()](usage/DashPlatformProtocol.md)
+    - [dpp.dataContract](usage/dataContract.md)
+    - [dpp.document](usage/document.md)
+    - [dpp.identity](usage/identity.md)
```

Comment:
This link doesn't work due to incorrect file name (link to identity also not working other places for the same reason). There seem to be some inconsistent use of case in file names. Personally I would prefer they start with lowercase, but as long as they're consistent we should be okay.

Reply:
Any incorrect file name is a bug, so I'm double checking this. 
However, reasoning behind lowercase vs uppercase : upperCase (Identity, DashPlatformProtocol) refers to class we can instantiates, while lowercase are namespace. 

When you do 

```
const dpp = new DashPlatformProtocol()
``` 
You got facades that are accessibles : `dpp.dataContract` which are not `DataContract` (defined in primitives/DataContract). 
That is what I'm trying to get explicitly here. How should I fix that ?

Reply:
Hmm, apparently it is may be more complicated than just a broken link. :slightly_smiling_face: I'd say stick with your plan. For this specific case, the identity usage doc is named `docs/usage/Identity.md` while the link is trying to use `docs/usage/identity.md`. If I'm understanding you, the file needs to be renamed to `docs/usage/identity.md`, correct?

Reply:
It was an issue indeed ! Thank you for noticing, fixed :)